pkg/apis/core/types.go

```diff
@@ -354,7 +354,7 @@ type PersistentVolumeSource struct {
 	// FlexVolume represents a generic volume resource that is
 	// provisioned/attached using an exec based plugin. This is an alpha feature and may change in future.
 	// +optional
-	FlexVolume *FlexVolumeSource
+	FlexVolume *FlexPersistentVolumeSource
```

Comment:
*Nit - do we need to repeat the Persistent in FlexPersistentVolumeSource? I like FlexVolumeSource